topology/probes/blockdev/metadata.go

```diff
@@ -0,0 +1,98 @@
+//go:generate go run github.com/skydive-project/skydive/scripts/gendecoder -package github.com/skydive-project/skydive/topology/probes/blockdev
```

Comment:
Please add the metadata_easyjson.go and metadata_gendecoder.go files to the repository

Reply:
ok